src/models/Product.ts

```diff
@@ -49,7 +49,13 @@ export default class Product extends Model {
     type: DataType.DOUBLE(3, 1),
     allowNull: false
   })
-  rating!: string;
+  rating!: number;
+
+  @Column({
```

Comment:
Do you use this to check if the product is limited? This feature is for marketing, it shouldn't be real. You can add a flag `isLimited` instead. So the admin can control that.